charts/metallb/Chart.yaml

```diff
@@ -20,10 +20,10 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 # NOTE: this value is updated by the metallb release process
-version: 0.0.0
+version: 0.12.2
```

Comment:
I'm not sure this is supposed to be updated in PRs. @fedepaol doesn't the release process increment this on release?

Reply:
Yep we do that here https://github.com/metallb/metallb/blob/4b97250b28956ece4a1e5ef882d24bffd5395f83/tasks.py#L627 (I even think you implemented it :P )